basic-analysis/basic-analysis.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://assets-global.website-files.com/63c799a4fa41261ff1f92fd4/64213c94885ad9858a1c2c2b_01-favicon%20(1).png" rel="shortcut icon" type="image/x-icon"/>
+  <link href="https://assets-global.website-files.com/63c799a4fa41261ff1f92fd4/64213c9730d27c215422b7f9_01-favicon%20(2).png" rel="apple-touch-icon"/>
+  <title>BlendVision One Web Player Samples | Basic Analysis</title>
+  <style>
+    * {
+      box-sizing: border-box;
+    }
+    html, body {
+      margin: 0;
+      font-size: '16px';
+    }
+    .player { 
+      height: 100vh;
+    }
+    .on-device .player { 
+      height: calc(100vh - 50px);
+    }
+    @media screen and (orientation: portrait) {
+      .device-ios .player { 
+        height: calc(100vh - 114px);
+      }
+    }
+  </style>
+</head>
+<body>
+  <div id='my-player'></div>
+  <script src="https://unpkg.com/@blendvision/player@2.10.0-canary.0"></script>
```

Comment:
Need to replace to formal release version `https://unpkg.com/@blendvision/player@2.10.0`

Reply:
fix in [0b87f5b](https://github.com/BlendVision/web-player-samples/pull/4/commits/0b87f5b2aaed66aec20ff6298d6a826c329c0a0c)